src/tools/rbd/Utils.h

```diff
@@ -52,6 +52,10 @@ static const std::string RBD_DIFF_BANNER_V2 ("rbd diff v2\n");
 #define RBD_EXPORT_IMAGE_STRIPE_COUNT	'C'
 #define RBD_EXPORT_IMAGE_END		'E'
 
+#define RBD_EXPORT_IMAGEMETA_KEY		'K'
+#define RBD_EXPORT_IMAGEMETA_VALUE		'V'
+#define RBD_EXPORT_IMAGEMETA_END		'D'
+
```

Comment:
I prefer to put the image-meta related into header. That means we have only two sections in the exported file, HEADER and DIFFs. So we can put the metadatas into HEADER section.